src/main/scala/nl/knaw/dans/easy/umd/Command.scala

```diff
@@ -33,31 +33,29 @@ object Command {
     val ps = cmd.parse(args)
     FedoraRequest.setDefaultClient(new FedoraClient(ps.fedoraCredentials))
     run(ps) match {
-      case s: Success[List[Any]] if s.get.nonEmpty => () // logged by run
-      case f: Failure[_] => log.error(s"failed", f.exception)
-      case s: Success[_] => log.info("success")
+      case Success(()) => log.info("success")
+      case Failure(e) => log.error("failed", e)
```

Comment:
@rvanheest 

I'm afraid we are now repeating the stack trace of `recoverWith`. I'm reluctant to drop the `e` argument there in case some day run is executed by akka or something like ingest-flow. Not sure it is safe to drop it here.